src/mirth/codegen.mth

```diff
@@ -113,6 +113,9 @@ def(c99-emit-header!, +IO,
     "#define STACK_SIZE 2000" ;
     "static volatile usize sc = STACK_SIZE;" ;
     "static volatile i64 stack[STACK_SIZE] = {0};" ;;
+
+    "int global_argc;" ;
```

Comment:
global variables are icky but I couldn't think of another way to get these from `main`

Reply:
👍 this is super reasonable for now!